.circleci/config.yml

```diff
@@ -205,35 +203,6 @@ workflows:
             - init
           workspace_path: ../workspace
 
-      # Trigger all workflows for branches that match regex
-      - continuation/continue:
```

Comment:
so youre saying this happens somewhere else now?

Reply:
Yup! It's handled by the path-filtering step above